GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1521,6 +1490,69 @@ static IEnumerable<ObjectId> TryGetParents(GitModule module, FilterInfo filterIn
             }
         }
 
+        public string? GetRevisionFileName(string path, string revisionId)
+        {
+            const string startOfObjectId = "????";
+
+            GitArgumentBuilder args = new("log")
+            {
+                // --name-only will list each filename on a separate line, ending with an empty line
+                // Find start of a new commit with a sequence impossible in a filename
+                $"--format=\"{startOfObjectId}%H\"",
+                "--name-only",
+                "--follow",
+                "--diff-merges=separate",
+                FindRenamesAndCopiesOpts(),
```

Comment:
reflog if configured, also for the "standard" call
Add a comment where this is used (may be put elsewhere if you refactoe according to RussKie suggestions

Reply:
I am not sure what you mean by the "standard" call. This method is only used in the File History dialog and because the actual revision ID is provided, we don't need to specify `--reflog` (or `--all` for that matter).

Reply:
You can still activate Reflog in FileHistory.

But this is just FileHistory, let us just try to finalize this.